fe/fe-core/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -1394,7 +1397,7 @@ private void getNewImage(Pair<String, Integer> helperNode) throws IOException {
                 MetaHelper.complete(filename, dir);
             }
         } catch (Exception e) {
-            return;
+            throw new IOException(e);
```

Comment:
Could you please add some Error Log in here.

Reply:
The error log will be printed outside